subjects/java/checkpoints/factory-blueprint/README.md

```diff
@@ -0,0 +1,62 @@
+## Factory Blueprint
+
+### Instructions
+
+You are given an incomplete Factory design pattern implementation with some incorrect parts. Complete and fix the class to demonstrate your understanding of how the Factory design pattern works.
+
+The method `showDetails` should print the class name 'ConcreteProductA' for `ConcreteProductA` and so on.
```

Comment:
This is very vague. Maybe add an example in the `ExerciseRunner`? Also, I know that the tests are not testing the output, but a learner during the exam doesn't know this and might "paralize" because of the missing information and not knowing how to implement this method

Reply:
is it better now ?